tox.ini

```diff
@@ -1,23 +1,24 @@
 [tox]
-envlist = py27,py27-simplejson,py34
+envlist = {py27,py35,py36}-{default,simplejson},pre-commit,cover,docs
```

Comment:
It's only the list of the available tox environemnts.
If someone runs ``tox`` it will run also docs, but in general the usage pattern is ``TOXENV=xxx tox`` or ``tox -e xxx``.